sonar-java-plugin/pom.xml

```diff
@@ -164,7 +229,7 @@
             <configuration>
               <rules>
                 <requireFilesSize>
-                  <maxsize>20000000</maxsize>
+                  <maxsize>30000000</maxsize>
```

Comment:
:eyes:

Reply:
Reverted. 
It was a change made during testing, and I forgot to revert it.